website/src/components/Chat/ChatListBase.tsx

```diff
@@ -79,6 +79,7 @@ export const ChatListBase = memo(function ChatListBase({
         bg: "blackAlpha.400",
       }}
       {...props}
+      zIndex="var(--chakra-zIndices-base)"
```

Comment:
why do we need zIndex on this? Isn't it is the last layer by default?

Reply:
You are right, I've redacted the said line.